src/lib.rs

```diff
@@ -531,6 +541,8 @@ impl Default for WindowAttributes {
             title: "winit window".to_owned(),
             maximized: false,
             fullscreen: None,
+            center_window: true,
```

Comment:
I don't think it's right to set `center_window: true` as the default behavior. Let the platform decide where to put it. This is better off as opt-in behavior.